### PR TITLE
fix/PL 2577/refactor tests

### PR DESCRIPTION
**Pull request by davidmdm**
- **wip**
- **fix(PL-2557): support chart with file uris**
- **fix(PL-2557): make test_data/chart/base basic**
- **fix(PL-2557): move catalog loading tests into catalog pkg**
- **fix(PL-2577): use local chart for render diff test**

### CODE REVIEW

File: pkg/catalog/catalog_test.go [hunk 1]
```diff
@@ -1,58 +1,88 @@
...
 }
+
+func TestFreeformEnvsAndReleasesLoading(t *testing.T) {
+	catalogDir, err := filepath.Abs("testdata/freeform")
+	assert.NoError(t, err)
+
+	cat, err := Load(catalogDir, nil)
+	assert.NoError(t, err)
+
+	// Environments
```
Comment (silphid) [inline pkg/catalog/catalog_test.go:50]: All those comments... 😵‍💫

File: pkg/catalog/testdata/broken-chart-ref-environment/environments/testing/releases/test-release.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      container:
+        port: 3000
+        livenessProbe:
+          httpGet:
+            path: /napi/healthz
+            port: 3000
+      deployment:
+        replicas: 1
+      service:
+        targetPort: 3000
```
Comment (silphid) [inline pkg/catalog/testdata/broken-chart-ref-environment/environments/testing/releases/test-release.yaml:27]: Maybe strip away all text that possibly can be removed in all files to simplify and emphasize objective of test?

**Review by silphid — APPROVED** [review]
Maybe just strip all `testdata` files to their bare minimum required by each test to reduce noise and emphasize each test's objective?

File: pkg/catalog/catalog_test.go (hunk 1 above)
Comment (davidmdm) [reply pkg/catalog/catalog_test.go:50]: They're historical artifacts !

**Pull request merged**